README.adoc

```diff
@@ -30,7 +30,9 @@ and is built by them.
 For now, issue:
 [source, bash]
 ----
-go test ./...
+kafka-topics --delete --zookeeper localhost:2181 --topic test-topic
```

Comment:
Maybe all of this should be part of a Makefile, under `make test`.
This to reduce differences between local and CI integration. CI shall just leverage make

Reply:
Yes, that's fair. I'll rework.